frontend/src/components/projects/Projects.tsx

```
@@ -5,16 +5,22 @@ import { useAxios, getSeason } from '../HelperFunctions';
-const Projects = (props: any) => {
+interface ProjectsProps {
+  isFeatured: boolean;
+  containerClassName?: string;
+  showSectionTitle?: boolean;
+}
+
+const Projects = ({ isFeatured, containerClassName, showSectionTitle = true }: ProjectsProps) => {
   //check which type of projects were rendering
-  const res = props.isFeatured == true
+  const res = isFeatured == true
     ? useAxios(
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)